firefox_puppeteer/api/places.py

```diff
@@ -108,29 +108,31 @@ def restore_default_bookmarks(self):
 
     def remove_all_history(self):
         """Removes all history items."""
-        try:
-            self.marionette.execute_async_script("""
-                Components.utils.import("resource://gre/modules/Services.jsm");
+        with self.marionette.using_context('chrome'):
 
```

Comment:
nit: please remove this extra empty line and we will be good to get this PR merged. Thanks.